Sources/AppStoreConnectCLI/Services/Operations/ListUsersOperation.swift

```diff
@@ -6,42 +6,51 @@ import struct Model.User
 
 struct ListUsersOperation: APIOperation {
 
-    private let endpoint: APIEndpoint<UsersResponse>
+    typealias Filter = ListUsers.Filter
+    typealias Limit = ListUsers.Limit
 
-    init(options: ListUsersOptions) {
-        let include = options.includeVisibleApps ? [ListUsers.Include.visibleApps] : nil
-
-        let limit = [
-            options.limitUsers.map(ListUsers.Limit.users),
-            options.limitVisibleApps.map(ListUsers.Limit.visibleApps)]
-            .compactMap { $0 }
-            .nilIfEmpty()
+    var limit: [ListUsers.Limit]? {
```

Comment:
Why not use the typealias?
```suggestion
    var limit: [Limit]? {
```